plugins/UM3NetworkPrinting/resources/qml/PrintJobInfoBlock.qml

```diff
@@ -374,28 +523,141 @@ Item
                 PrintCoreConfiguration
                 {
                     id: leftExtruderInfo
-                    width: Math.round(parent.width / 2)
+                    width: Math.round(parent.width / 2) * screenScaleFactor
                     printCoreConfiguration: printJob.configuration.extruderConfigurations[0]
                 }
 
                 PrintCoreConfiguration
                 {
                     id: rightExtruderInfo
-                    width: Math.round(parent.width / 2)
+                    width: Math.round(parent.width / 2) * screenScaleFactor
                     printCoreConfiguration: printJob.configuration.extruderConfigurations[1]
                 }
             }
 
         }
-
+        }
         Rectangle
         {
+            height: cardHeight * screenScaleFactor
             color: UM.Theme.getColor("viewport_background")
-            width: 2
+            width: 2 * screenScaleFactor
             anchors.top: parent.top
-            anchors.bottom: parent.bottom
             anchors.margins: UM.Theme.getSize("default_margin").height
             anchors.horizontalCenter: parent.horizontalCenter
         }
+
+        // Alert / Configuration change box
+        Rectangle
+        {
+            height: alertHeight() * screenScaleFactor
+
+            anchors.left: parent.left
+            anchors.right: parent.right
+            anchors.bottom: parent.bottom
+
+color: "#ff00ff"
```

Comment:
Codestyle: indentation.
Also, that color should probably be themeable, or is this one of your "Rectangle-that-should-be-an-Item" elements with a debug color?